### ISSUE TITLE
`json-schema` Endpoint: Single Quotes in the "description" field of a JSON Schema property cause errors.

### ISSUE THREAD

**Issue by kidrecursive** [stargate/stargate#1290]
When creating a JSON schema that looks like this: 
```json
{

  "$id": "https://example.com/person.schema.json",
  "$schema": "https://json-schema.org/draft/2020-12/schema",
  "title": "Person",
  "type": "object",
  "properties": {
    "firstName": {
      "type": "string",
      "description": "The person's first name."
    },
    "lastName": {
      "type": "string",
      "description": "The person's last name."
    },
    "age": {
      "description": "Age in years which must be equal to or greater than zero.",
      "type": "integer",
      "minimum": 0
    }
  }
}
```
The API returns a 500. When escaping the single quote, the API returns a 400. The request was issued via the Swagger UI.

**Comment by dougwettlaufer** [comment]
@kidrecursive do you have the response message from the 500 and 400?

**Comment by kidrecursive** [comment]
Sure thing:

400 
```json
{
  "description": "Malformed JSON schema provided.",
  "code": 400
}
```

500
```
{
  "description": "Server error: org.apache.cassandra.stargate.exceptions.SyntaxException: line 1:262 mismatched input 's' expecting EOF (....crw_test WITH comment = '{\"schema\":{\"$id\":\"https://example.com/person.schema.json\",\"$schema\":\"https://json-schema.org/draft/2020-12/schema\",\"title\":\"Person\",\"type\":\"object\",\"properties\":{\"firstName\":{\"type\":\"string\",\"description\":\"The person'[s]...)",
  "code": 500
}
```

**Comment by tatu-at-datastax** [comment]
Sounds like there might be two issues: first, failure should be 4xx not 500 (for the first case) if there's an issue with user-provided input. And second, something about escaping at CQL level may be off (error does not come from JSON parser I think, but from underlying CQL statement).

**Comment by ivansenic** [comment]
This is a Swagger issue, just tested with your schema and command like:

```
curl -X PUT -d @schema.json -H 'content-type: application/json' -H 'X-Cassandra-Token: ..' "http://127.0.0.4:8082/v2/namespaces/ns/collections/test/json-schema"
```

**Comment by kidrecursive** [comment]
Heya, getting the same thing from `cURL`, `Postman` and `@astrajs`, against Astra and Stargate (v1.0.34)

**Comment by ivansenic** [comment]
@kidrecursive OK I am investigating..

**Comment by ivansenic** [comment]
@kidrecursive You were correct, this is a bug indeed. I am sorry that my previous comment was suggesting it's OK. The problem itself is not in the document API, but rather in the query builder. I have created another ticket here for sake of correctness and we will jump to provide a fix asap.

See https://github.com/stargate/stargate/issues/1293 for details.

**Comment by ivansenic** [comment]
Fixed in #1294, I extended our schema integration tests to cover this case as well.

**Comment by kidrecursive** [comment]
Thanks @ivansenic!
